app/src/main/java/com/example/drawingactivity/CustomColorViewModel.kt

```diff
@@ -0,0 +1,35 @@
+package com.example.drawingactivity
+
+import android.graphics.Color
+
+class CustomColorViewModel(
+
+) {
+    var onUpdate: ((ViewStates) -> Unit)? = null
+
+    data class ViewStates(
+        var primaryColor: Int,
+    )
+
+    private var primaryColor = -0x10000
```

Comment:
Why this? What color do you want? Just use hex like 0xff000000

Reply:
Adjusted Colors to be more sensible.
Array is now a list